fuzzers/044-clk-bufg-pips/Makefile

```diff
@@ -0,0 +1,61 @@
+export FUZDIR=$(shell pwd)
```

Comment:
Normal way to do this is;
```make
mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
current_dir := $(notdir $(patsubst %/,%,$(dir $(mkfile_path))))
```

Reply:
That might be the normal way, but it seems less readable to me.